website/docs/d/tg_connection_prefix_filter.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+
+subcategory: "Transit Gateway"
+layout: "ibm"
+page_title: "IBM : tg_connection_prefix_filter"
+description: |-
+  Manages IBM Cloud Infrastructure Transit Gateway Connection Prefix Filter.
```

Comment:
Kindly change description..
Since it is datasource, it cannot manage filter.. 
Add something like `Reads` / `Gets Information of` IBM Cloud Infrastructure Transit Gateway Connection Prefix Filter.

Same change to other datasource